plan-patches/cf-azure/ops/rename-network-and-deployment.yml

```diff
@@ -0,0 +1,252 @@
+---
+# Network name
+- type: replace
+  path: /instance_groups/name=smoke-tests/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=consul/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=nats/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=doppler/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=adapter/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=database/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=diego-api/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=uaa/networks/name=default/name
+  value: ((network_name))
+
+# - type: replace
```

Comment:
Could you please remove these comment lines if they are not required?

Reply:
changed. thanks.